DynmapCore/src/main/resources/texture_1.txt

```diff
@@ -3201,3 +3201,538 @@ block:id=%attached_melon_stem,patch0=0:melon_stem,patch1=0:attached_melon_stem,b
 block:id=%pumpkin_stem,patch0=0:pumpkin_stem,blockcolor=foliagebiome,transparency=TRANSPARENT,stdrot=true
 # Melon stem
 block:id=%melon_stem,patch0=0:melon_stem,blockcolor=foliagebiome,transparency=TRANSPARENT,stdrot=true
+<<<<<<< HEAD
+=======
```

Comment:
These kinds of lines are merge markers. Basically, you performed a merge at some point and Git couldn't figure out how to automatically merge the changes, so it put these markers there to tell you that you need to do it manually.

I'd suggest just checking out the upstream master branch, changing the one line, committing it, and pushing that commit to your `brick_fix` branch (using the `--force` flag).

Or if you'd like, I can open a different PR with the changes so you don't have to keep messing around with Git.